src/components/tasklist/Tasklist.js

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+
+import './tasklist.scss';
+
+import { FaTrashAlt, FaBatteryThreeQuarters, FaCheck } from 'react-icons/fa';
+
+class Tasklist extends Component {
+
+    userId = 3;
+
+    noResults = "No Results";
+
+    state = {
+        todos: [],
+        displayFilter: []
+    };
+
+    componentDidMount() {
+        fetch(`https://jsonplaceholder.typicode.com/todos?userId=${ this.userId }`)
```

Comment:
1) need to place it to separate service
2) need to use project server

Reply:
Not agree with that. It is will be in future, now it's according to homework from these slides
[https://docs.google.com/presentation/d/1vFkwy7f0xNBOXZTl7QQF2vnEOPd7izQjFjoPUYgFNhg/edit#slide=id.g3295adaefa_0_28](url)
[https://docs.google.com/presentation/d/1vFkwy7f0xNBOXZTl7QQF2vnEOPd7izQjFjoPUYgFNhg/edit#slide=id.g4a54792b90_0_0](url)
[https://docs.google.com/presentation/d/1vFkwy7f0xNBOXZTl7QQF2vnEOPd7izQjFjoPUYgFNhg/edit#slide=id.g257752e613_0_0](url)

Reply:
Ok, if it's really difficult for you)

Reply:
@iliatcymbal Create services not difficult, but you server have another data structure. Component should be rebuild after get data from your server.